NEWS.md

```diff
@@ -1,21 +1,22 @@
 Version 3.9 of OpenDDS.
+-------------------------------------------------------------------------------
 
 - Updated how DataWriter QoS is interpreted by internal data structures:
-  - RESOURCE_LIMITS.max_samples_per_instance controls total samples that can be
+  - RESOURCE\_LIMITS.max\_samples\_per\_instance controls total samples that can be
     stored per instance, independent of HISTORY
   - HISTORY.depth controls number of samples (per instance) that will be made
     available to late-joining readers if DURABILITY is enabled
-    - KEEP_ALL history uses max_samples_per_instance as its depth
+    - KEEP\_ALL history uses max\_samples\_per\_instance as its depth
   - WriteDataContainer is more eager to remove samples that are no longer needed
   - WriteDataContainer will not remove samples that are required to implement
     DURABILITY, hence if RELIABILITY is enabled this can block a write() just
-    like other "no resource available" scenarios
+    like other “no resource available” scenarios
```

Comment:
The quotes here look odd... I'm not sure if it's just how GitHub's web UI is rendering them.  Check if they are basic ASCII " characters.